test/modern_bpf/test_suites/syscall_exit_suite/creat_x.cpp

```diff
@@ -0,0 +1,116 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_creat
+
+#if defined(__NR_fstat) && defined(__NR_unlinkat)
+TEST(SyscallExit, creatX_success)
+{
+	auto evt_test = new event_test(__NR_creat, EXIT_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL  ===========================*/
+
+	const char* path = "tmp_file";
+	mode_t mode = S_IRWXU;
+	int fd = syscall(__NR_creat, path, mode);
+	assert_syscall_state(SYSCALL_SUCCESS, "creat", fd, NOT_EQUAL, -1);
+
+	/* Call `fstat` to retrieve the `dev` and `ino`. */
+	struct stat file_stat;
+	assert_syscall_state(SYSCALL_SUCCESS, "fstat", syscall(__NR_fstat, fd, &file_stat), NOT_EQUAL, -1);
+	uint32_t dev = (uint32_t)file_stat.st_dev;
+	uint64_t inode = file_stat.st_ino;
+
+	/* Remove the file. */
+	close(fd);
```

Comment:
minor question: You typically check for __NR__xxx for syscalls being used. Not sure if you like to add a define for close as well. So far I have seen that for the close bpf test case only.

Reply:
Yeah that's a good point, i will add it, we can also check if the syscall is successful like for the other cases with 
`assert_syscall_state(SYSCALL_SUCCESS,...)` but I'm not sure to want a test failure if the clone fails :thinking: WDYT? BTW i will surely add the define :)

Reply:
I added the `defines` but not checked for their results since probably we don't want a test failure in case we are not able to correctly close the file descriptor :thinking:

Reply:
I thought in the same direction with having the `close()` for clean-up rather than actual testing. It might be worth in some special cases when `write` is used and `close` might report an outstanding write error.

Reply:
Agree on that :+1: